app/scripts/controllers/authentication/authentication-controller.ts

```diff
@@ -152,6 +190,10 @@ export default class AuthenticationController extends BaseController<
     return profile;
   }
 
+  public isSignedIn(): boolean {
+    return this.state.isSignedIn;
```

Comment:
Dev Note - this is so state can be exposed through the messaging system. If we need to expose more state for other controllers we can widen this (or create new methods)